src/core/DSPCoreDataModel.ttl

```diff
@@ -351,17 +413,54 @@ DSPCore:Sequencing
   a owl:Class ;
   rdfs:label "Sequencing" ;
   rdfs:subClassOf DSPCore:Activity ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty DSPCore:usesSample ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:onProperty <http://www.w3.org/ns/prov#generated> ;
+      owl:someValuesFrom DSPCore:SequenceFile ;
+    ] ;
   skos:prefLabel "Sequencing" ;
 .
 DSPCore:UserRestriction
   a obo:DUO_0000026 ;
   rdfs:label "User Restriction" ;
   skos:prefLabel "User Restriction" ;
 .
+DSPCore:VariantCall
+  a owl:Class ;
+  rdfs:comment "Use closeMatch to identify the called variant in other systems.  For example HGMD, ClinVar, rsID." ;
+  rdfs:label "VariantCall" ;
+  rdfs:subClassOf obo:OBI_0001364 ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:minCardinality "1"^^xsd:nonNegativeInteger ;
+      owl:onProperty skos:closeMatch ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:onProperty DSPCore:hasLocation ;
+      owl:someValuesFrom DSPCore:ChromosomalLocation ;
+    ] ;
+  owl:equivalentClass [
+      a owl:Restriction ;
+      owl:onProperty <http://www.w3.org/ns/prov#used> ;
+      owl:someValuesFrom DSPCore:VariantCallSet ;
+    ] ;
+  skos:prefLabel "VariantCall" ;
+.
 DSPCore:VariantCallSet
   a owl:Class ;
   rdfs:label "VariantCallSet" ;
   rdfs:subClassOf DSPCore:File ;
+  owl:equivalentClass [
```

Comment:
The relationship between `DSPCore:VariantCall` and `DSPCore:VariantCallSet` is many-to-many, yes?

Reply:
yes.  An aligned sequence may use different Variant Callers or different versions of Variant Callers to produce multiple vcf or gvcf files (or maybe some new format).  The (g)vcf will identify many variants.  Additionally different filters can be used to create subsets of the variants; they shouldn't be different, but rather proper subsets.